expression/builtin_time.go

```diff
@@ -1770,6 +1770,52 @@ func getTimeZone(ctx context.Context) *time.Location {
 	return ret
 }
 
+// isDuration returns a boolean indicating whether the str is the format of duration.
+func isDuration(str string) bool {
+	const durationArgReg = `^(|[-]?)(|\d{1,2}\s)(\d{2,3}:\d{2}:\d{2}|\d{1,2}:\d{2}|\d{1,6})(|\.\d*)$`
```

Comment:
And add comment for it, better with a example: "xx:xx:xx", otherwise, nobody can understand this regexp.